src/searching.md

```diff
@@ -405,5 +409,5 @@ Note and card IDs can be found in the [card info](stats.md) dialog in the
 browser. These searches may also be helpful when doing add-on
 development or otherwise working closely with the database.
 
-Object IDs will not work in the mobile clients, and are not intended to
+Object IDs will not work in AnkiMobile, and are not intended to
```

Comment:
Is this still true for AnkiMobile? AnkiDroid supports it.

Reply:
They've been supported for some time, we could probably just drop this paragraph.